src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/EnforcementAction.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.Where;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@EntityListeners(AuditingEntityListener.class)
+@Entity
+@Table(name = "R_ENFORCEMENT_ACTION")
+@Where(clause = "SELECTABLE = 'Y'")
```

Comment:
Hmm, not sure about this. I think SELECTABLE means "is this enforcement available in the UI" which is different to what you need. If a contact is related to a enforcement action that has been deprecated then it should still be returned right?

Reply:
Yes, thats correct. Fixed